10-icmpv6-error/10-icmpv6-error.md

```diff
@@ -0,0 +1,188 @@
+## Goal: Check single-hop ICMP connectivity over IPv6
+
+*Note: This task requires to send some very specificly formated IPv6 packets. We
+recommend to use [scapy] for the construction of those packets in those tests.*
+
+Task #01 - Destination unreachable - no route to destination
+============================================================
+### Description
+
+Solicit **Destination unreachable - no route to destination** by sending a UDP
```

Comment:
It is a bit misleading to have mentions to UDP in an ICMPv6 test. I think it should rather be `ICMPv6EchoRequest` instead of `UDP` here and in other places, no?

Reply:
a) I don't really get what is misleading about this? These are tests to trigger ICMPv6 error messages. What kind of packet triggers them should be irrelevant (and in fact IMHO it should (but must not) at least for the tests be disjunct from ICMP so using e.g. UDP helps with that).
b) From a practical standpoint ICMPv6EchoRequest has the disadvantage that we can't easily register to it (it would require a dedicated command i.e. test application). `udp server` is up and ready in `gnrc_networking`. Just listening for an ICMPv6EchoReply could also be viable, but it gives no guarantee on which node it was handled (from a pure testing perspective of course were no assumptions can be made, in practice it should of course be the node targeted).

Reply:
Also: ICMPv6 has no ports, so testing Task 5 without UDP would be quite impossible ;-).